### PR TITLE
Added cerberus testnet

### PR DESCRIPTION
**Pull request by cerberus-zone**
Test Result:
- [x ] Connect wallet, check if address is correct? 
- [ x] Transfer
- [ x] Delegate
- [ x] Redelegate
- [x ] Unbond
- [ x] withdraw Validator's Commission
- [ x] withdraw Rewards

### CODE REVIEW

**Comment by liangping** [comment]
HI,  Thanks for PR. Currently the testnet is down.  I will try to launch it soon.

**Comment by joshlopes** [comment]
There is a lot of changes in this PR that i don't think should be here.

File: .eslintrc.js [hunk 1]
```diff
@@ -3,19 +3,27 @@ module.exports = {
...
-    'no-console': process.env.NODE_ENV === 'production' ? 'error' : 'off',
-    'no-debugger': process.env.NODE_ENV === 'production' ? 'error' : 'off',
-
-    semi: ['error', 'never'],
-    'max-len': 'off',
-    'linebreak-style': 'off',
-    camelcase: ['error', { properties: 'never', ignoreDestructuring: true, ignoreImports: true }],
-    'arrow-parens': ['error', 'as-needed'],
-    'vue/multiline-html-element-content-newline': 'off',
+    /* eslint quotes: ["off", "double"], curly: 2 */
```
Comment (liangping) [inline .eslintrc.js:11]: almost many changes are made by these changes. This really should NOT be include in this PR. 
FYI. testnet should added on `testnet` branch

Comment (cerberus-zone) [reply .eslintrc.js:11]: Thanks, for the call out. We forgot we had done some testing. We will remove the PR and submit a new one.

**Comment by cerberus-zone** [comment]
Closing PR and will be submitting a new one.

**Pull request closed without merging**